src/Carbon/Traits/Units.php

```diff
@@ -42,7 +42,7 @@ public function addRealUnit($unit, $value = 1)
             // @call addRealUnit
             case 'micro':
 
-            // @call addRealUnit
+                // @call addRealUnit
```

Comment:
This should not be indented because it belongs to the case below and not above.

Reply:
CS gives an error when it is as your suggested.

![image](https://github.com/user-attachments/assets/438baf4b-379e-47d5-9f0e-9597fa48c219)

Reply:
Fixed.

https://github.com/CarbonPHP/carbon/pull/13/commits/dc27804306d1197bc5f8f46f298acf13fa35ed7b